### ISSUE TITLE
cannot click on duo authentication button

### ISSUE THREAD

**Issue by Dooruk** [browsh-org/browsh#389]
Hi,

I would like to access my google drive to download some folders remotely, I am stuck at the authentication step. I can't click on the DUO link (green buttons in the screenshot). I tried zooming in, switching between screens but no avail. I am able to click on other webpages. 

![ss](https://user-images.githubusercontent.com/38666458/136075257-83b2e46f-add9-41b1-826c-83611cf3e4cb.png)

Thanks!

**Comment by Dooruk** [comment]
Yes, I am able to click links on other webpages, just not this particular one.

**Comment by martin-braun** [comment]
I have the same problems with the popups on Google and YouTube when you first visit those pages on a fresh profile: 

![Screen Shot 2023-01-27 at 6 56 55 PM](https://user-images.githubusercontent.com/24259245/215160017-de785e92-f368-4856-92dd-93d8efbb81be.png)

![Screen Shot 2023-01-27 at 6 57 22 PM](https://user-images.githubusercontent.com/24259245/215160038-5012ebcc-49ef-41d4-b0c3-e2fd6c083212.png)

These buttons can't be clicked ...

**Comment by tombh** [comment]
I wonder if this is a security issue. For instance maybe Google prevents the automated clicks from webextensions like the one Browsh uses to forward clicks from the terminal?
